docs/example-data.json

```diff
@@ -188,7 +187,9 @@
         "current_projector_$_ids": [],
         "vote_delegated_vote_$_ids": [],
         "vote_delegated_$_to_id": [],
-        "vote_delegations_$_from_ids": []
+        "vote_delegations_$_from_ids": [],
+
+        "meeting_ids": [1]
```

Comment:
Isn't it a calculated field based on group-membership? Then I would expect to be a group set and this one calculated

Reply:
type, fixed. Thanks!